### PR TITLE
feat: Rewrite Nerdpack permissions to clarify

### PR DESCRIPTION
**Pull request by alexronquillo**
## Description

This doc is highly confusing for many users and Relics. This rewrite aims to address the confusing issues and clarify Nerdpack permissions as best as possible.

### CODE REVIEW

File: src/markdown-pages/build-apps/permission-manage-apps.mdx [hunk 1]
```diff
@@ -13,41 +13,124 @@ tags:
...
 
-Basic users can build and serve their own Nerdpacks locally. They can also use public Nerdpacks that meet **both** of the following criteria:
+## Capabilities at a glance
 
-* New Relic must have specifically allowed basic users to use the Nerdpack
-* Someone else on the basic user's account must have already subscribed to the Nerdpack
+Your ability to manage and use Nerdpacks depends on your user's:
 
-Nerdpacks that basic users can use are rare.
+- Model
```
Comment (mmfred) [inline src/markdown-pages/build-apps/permission-manage-apps.mdx:26]: Here, you might say something like, " Model (original or New Relic One model)"
I recommend this because when I saw Model, I was confused, and I'm used to this distinction. If that's too wordy, you could make it "User model" -- I know that introduces a little redundancy with the lead in.

File: src/markdown-pages/build-apps/permission-manage-apps.mdx [hunk 2]
```diff
@@ -13,41 +13,124 @@ tags:
...
-* A Nerdpack made by your team can only be subscribed to from the master account that was used to publish it, or from its sub-accounts. This means that, if the Nerdpack needs to be available across your organization, you may need a New Relic admin to deploy it.
\ No newline at end of file
+## Nerdpack permissions: Original user model
+
+On both the [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/) and the [New Relic One user model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/), user permissions differ between types and roles. In this section, you learn the differences between how basic users, full users, and admins can use Nerdpacks in the original user model.
+
+### Basic users
+
+Basic users in the original user model are limited in their Nerdpack capabilities. They can only create and serve Nerdpacks locally. To publish, tag, subscribe to, and use their Nerdpacks, a basic user must be upgraded to a full user. An admin user on their account can do this for them.
```
Comment (mmfred) [inline src/markdown-pages/build-apps/permission-manage-apps.mdx:70]: Can you make this "you" centered - "If you're a basic user, you have limited Nerdpack capabilites. You can only create... "

File: src/markdown-pages/build-apps/permission-manage-apps.mdx [hunk 3]
```diff
@@ -13,41 +13,124 @@ tags:
...
+
+<Callout variant="tip">
+
+While, in general, a basic user can't use Nerdpacks, even ones that their accounts create and publish, there are some Nerdlets that New Relic maintains that basic users can use. However, these are rare and there is currently no way to distinguish them in our web interface. And even if a basic user is allowed to use a Nerdlet in a Nerdpack, they are never able to use custom visualizations. To use custom visualizations, a user must be at least a full user.
```
Comment (mmfred) [inline src/markdown-pages/build-apps/permission-manage-apps.mdx:74]: Make "you" centered throughout. I'll stop making this comment :)

File: src/markdown-pages/build-apps/permission-manage-apps.mdx [hunk 4]
```diff
@@ -13,41 +13,124 @@ tags:
...
-## Role-related restrictions
+| | Basic user | Full user without the Nerdpack Manager add-on role | Full user with the Nerdpack Manager add-on role | Owner/Admin user |
+|:--|:-:|:-:|:-:|:-:|
+| Serve Nerdpacks | yes | yes | yes | yes |
+| Publish Nerdpacks | no | no | yes | yes |
+| Subscribe to Nerdpacks | no | no | yes | yes |
+| Tag Nerdpacks | no | no | yes | yes |
+| Use Nerdlets or visualizations created by the user's accounts | no | yes | yes | yes |
+| Use Nerdlets created by New Relic | no* | yes | yes | yes |
+| Use visualizations created by New Relic | no | yes | yes | yes |
```
Comment (JuliaNocera) [inline src/markdown-pages/build-apps/permission-manage-apps.mdx:44]: Since we capitalize `Nerdlets` should we also capitalize `Visualizations`?

File: src/markdown-pages/build-apps/permission-manage-apps.mdx [hunk 5]
```diff
@@ -13,41 +13,124 @@ tags:
...
+
+A full user with the **Nerdpack Manager** role can perform any of the Nerdpack capabilities and is, therefore, equivalent to an owner or admin within the context of Nerdpacks.
+
+### Owner/Admin users
+
+An owner or admin user can perform any of the Nerdpack capabilities, including creating, serving, publishing, tagging, subscribing to, and using any and all Nerdpack artifacts, both Nerdlets and visualizations, either built by New Relic or by their own accounts.
+
+## Nerdpack permissions: New Relic One user model
+
+On both the [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/) and the [New Relic One user model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/), user permissions differ between types and roles. In this section, you learn the differences between how basic users and full users can use Nerdpacks in the New Relic One user model.
```
Comment (mmfred) [inline src/markdown-pages/build-apps/permission-manage-apps.mdx:97]: It would be great to avoid the sort of clunky "In this section, you learn..." sentence. Even something simple like, "Read on for information about the differences between full and basic user capabilities..." You can find a nice way to provide this info.

File: src/markdown-pages/build-apps/permission-manage-apps.mdx [hunk 6]
```diff
@@ -13,41 +13,124 @@ tags:
...
+
+### Basic users
+
+Like in the original user model, basic users in the New Relic One user model are limited in their Nerdpack capabilities. They can only create and serve Nerdpacks locally. To publish, tag, subscribe to, and use their Nerdpacks, a basic user must be upgraded to a full user by an admin.
+
+<Callout variant="tip">
+
+While, in general, a basic user can't use Nerdpacks, even ones that their accounts create and publish, there are some Nerdlets that New Relic maintains that basic users can use. However, these are rare and there is currently no way to distinguish them in our web interface. And even if a basic user is allowed to use a Nerdlet in a Nerdpack, they are never able to use custom visualizations. To use custom visualizations, a user must be at least a full user.
```
Comment (mmfred) [inline src/markdown-pages/build-apps/permission-manage-apps.mdx:105]: Remove commas around "in general" for a little better flow.

File: src/markdown-pages/build-apps/permission-manage-apps.mdx [hunk 7]
```diff
@@ -13,41 +13,124 @@ tags:
...
+### Full users
+
+Full users in the New Relic One user model fit into two categories:
+
+- Those who have been granted the **Nerdpacks "modify"** privilege
+- Those who have not been granted the **Nerdpacks "modify"** privilege
+
+The **Nerdpacks "modify"** privilege is required to publish, subscribe to, and tag Nerdpacks. So, a full user who doesn't have the **Nerdpacks "modify"** privilege is only allowed to create and serve Nerdpacks locally and use Nerdpacks that their accounts have already been subscribed to.
+
+So, to publish, tag, or subscribe to your Nerdpack, an admin must grant you the **Nerdpacks "modify"** privilege.
```
Comment (mmfred) [inline src/markdown-pages/build-apps/permission-manage-apps.mdx:118]: the commas after a couple of "so" sentence starts aren't necessary.

**Review by mmfred — DISMISSED** [review]
Some suggestions for you.

File: src/markdown-pages/build-apps/permission-manage-apps.mdx (hunk 4 above)
Comment (alexronquillo) [reply src/markdown-pages/build-apps/permission-manage-apps.mdx:44]: Unfortunately not, at least not according to the naming guides I've been given. "Visualization" is a common noun, whereas "Nerdlets" is proper because it's a branded name.

File: src/markdown-pages/build-apps/permission-manage-apps.mdx (hunk 2 above)
Comment (alexronquillo) [reply src/markdown-pages/build-apps/permission-manage-apps.mdx:70]: Yeah, I was wondering if this might be raised 😄 I thought about this a lot and the reason I opted against it was because it feels more like a strictly generic doc (similar to [this section](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/#user-type)) as the audience isn't 100% clear. For example, I might be the basic user who came across an issue, so I'm looking it up. Or, I might be the basic user's admin looking this up on their behalf. I'm probably over-thinking this and could go either way... so yes, I'll change it 👍

File: src/markdown-pages/build-apps/permission-manage-apps.mdx (hunk 1 above)
Comment (alexronquillo) [reply src/markdown-pages/build-apps/permission-manage-apps.mdx:26]: Yep, originally, I had "User's" at the front of all of these list items, but felt it was too verbose. I think your parenthetical suggestion works.

**Comment by alexronquillo** [comment]
@mmfred Thanks for the review! I've submitted some commits addressing your feedback!

**Review by jaesius — APPROVED** [review]
